src/resources/views/crud/inc/form_fields_script.blade.php

```diff
@@ -0,0 +1,183 @@
+<script>
+    /**
+     * A front-end representation of a Backpack field, with its main components.
+     *
+     * Makes it dead-simple for the developer to perform the most common
+     * javascript manipulations, and makes it easy to do custom stuff
+     * too, by exposing the main components (name, wrapper, input).
+     */
+    class CrudField {
+        constructor(name) {
```

Comment:
I've just double-checked the selection logic and it still sounds good to me! Good job here @pxpm ! 👏👏👏

@promatik could you please also give it a look too? Maybe you spot red flags I missed, like you did with looking up through ancestors. Look at `constructor()` but also at `inputWrapper` and `mainInput` please.